pkg/policy/api/rule.go

```diff
@@ -252,3 +258,50 @@ type PortRuleHTTP struct {
 	// +optional
 	Headers []string `json:"headers,omitempty"`
 }
+
+// PortRuleKafka is a list of Kafka protocol constraints. All fields are
+// optional, if all fields are empty or missing, the rule does not have any
+// effect.
+type PortRuleKafka struct {
+	// APIVersion is the version matched against the api version of the
+	// Kafka message. It is always "0" or a string representing a
+	// positive integer.
+	//
+	// +optional
+	APIVersion string `json:"apiVersion,omitempty"`
+
+	// APIKey is a string matched against the key of a
+	// request, e.g. "Produce", "Fetch", "CreateTopic", "DeleteTopic", ...
```

Comment:
Echoing @tgraf’s comment, I suppose the value is case-insensitive? If so, specific that explicitly in this comment.

Reply:
@rlenglet I was trying to make this as a simple first step and hence took some assumptions. Let me make the changes and put them out for review.

Reply:
done